pkg/cli/importimage/importimage.go

```diff
@@ -207,6 +212,15 @@ func (o *ImportImageOptions) Validate() error {
 		return fmt.Errorf("valid reference policy values are source or local")
 	}
 
+	switch o.ImportMode {
+	case string(imagev1.ImportModeLegacy):
+	case string(imagev1.ImportModePreserveOriginal):
+	case "":
```

Comment:
Due to this assignment, there is no problem. But would it be better to set `ImportMode` to `Legacy` in this initialization https://github.com/openshift/oc/blob/d27e7c9efc6b2e1f6436d47eb4de2024e5a3d856/pkg/cli/importimage/importimage.go#L97 like we do for `ReferencePolicy`?

Reply:
That would seem to be the better place, but this is also done for `ReferencePolicy` just above. I'm wondering if this is needed for either of these.

Reply:
Ok, it seems this assignment is needed here regardless in the specific case that the user specifies `--reference-policy="" or --import-mode=""`.